src/api/wallet/WalletApiImpl.ts

```diff
@@ -248,7 +248,6 @@ export class WalletApiImpl implements WalletApi {
   private async _notifyListeners(): Promise<void> {
     await Promise.resolve()
     const walletInfo: WalletInfo = this._getWalletInfo()
-    console.table(walletInfo)
```

Comment:
But, but we never get to use `console.table` :cry:

Reply:
@Velenir I always know when u find some new thing when it pops up EVERYWHERE lol.

Reply:
I'm happy to keep it if we can manage to log it ONLY when it changes